wandb/env.py

```diff
@@ -293,10 +291,10 @@ def get_cache_dir(env=None):
     return val
 
 
-def get_use_v1_artifacts(env=None):
+def get_artifact_checksum_parallelism(env=None):
     if env is None:
         env = os.environ
-    val = env.get(USE_V1_ARTIFACTS, False)
+    val = env.get(ARTIFACTS_CHECKSUM_PARALLELISM, 8)
```

Comment:
We may want to default this to some factor of `multiprocessing.cpu_count()`

Reply:
Good idea.